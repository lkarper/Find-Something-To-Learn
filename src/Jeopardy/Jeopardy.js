import React, { Component } from 'react';

class Jeopardy extends Component {
    render() {
        return (
            <h2>Jeopardy!</h2>
        );
    }
}

export default Jeopardy;