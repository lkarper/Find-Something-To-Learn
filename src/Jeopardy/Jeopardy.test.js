import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import Jeopardy from './Jeopardy';

describe('Jeopardy component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
            <BrowserRouter>
                <Jeopardy />
            </BrowserRouter>, div);
        ReactDOM.unmountComponentAtNode(div);
    });
});