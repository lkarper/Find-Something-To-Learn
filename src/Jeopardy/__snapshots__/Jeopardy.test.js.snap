// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jeopardy component renders the UI as expected 1`] = `
<ContextProvider
  value={
    Object {
      "addToLearningList": [Function],
      "checkAnswerAndUpdateScore": [Function],
      "correct": false,
      "correctAnswersObject": Object {},
      "currentQuestion": 0,
      "gameType": "jeopardy",
      "goToNextQuestion": [Function],
      "goToWiki": [Function],
      "handleNewQuestionsJeo": [Function],
      "learningList": Object {},
      "page": "",
      "questions": Array [],
      "removeItemFromLearningList": [Function],
      "resetLearningList": [Function],
      "startNewGame": [Function],
      "totalQuestions": 0,
    }
  }
>
  <h2>
    Jeopardy!
  </h2>
  <JeopardyPlay />
</ContextProvider>
`;
